### PR TITLE
Feature/spell check

### LINKED ISSUE #88: Spell-check test
It would be nice if during the testing the manual got spell-checked...

I think the best way to do this would be programmatically, with something like [atom/node-spellchecker](https://github.com/atom/node-spellchecker), however the NPM installer for Mac is currently broken.

Alternatively, we could use [node-markdown-spellcheck](https://github.com/lukeapage/node-markdown-spellcheck), but this is only CLI based, so might not be as clean.

### PR DESCRIPTION
**Pull request by garethnunns**
Spell checker - run with `npm run spellcheck` from `./website`

Also added to `npm test` so runs on every PR too

Resolves #88

### CODE REVIEW

**Comment by garethnunns** [comment]
@Farrser and I made some calls on how things should be spelt based on the most common spelling in the software but feel free to suggest changes or add/remove items from .spelling

**Review by Farrser — CHANGES_REQUESTED** [review]

File: docs/titan-basics.md [hunk 1]
```diff
@@ -25,7 +25,7 @@ console normally.
 
 If an internal UPS is fitted (Sapphire Touch, Arena, optional on other
 consoles), on power loss the toolbar on the screen will turn orange and
-display how long you\'ve got before the console shuts down. A warning dialog
+display how long you've got before the console shuts down. A warning dialog
```
[the remark this answers is not in the document]
Comment (garethnunns) [reply docs/titan-basics.md:28]: Nope, no need https://www.markdownguide.org/basic-syntax/#characters-you-can-escape

File: website/versioned_docs/version-13.0/chases/chase-options.md [hunk 2]
```diff
@@ -49,7 +49,7 @@ Allows you to set a speed factor for the chase.\
 ### Speed
 Provides another way of setting chase speed.
 
-### XFade
+### Xfade
```
[the remark this answers is not in the document]
Comment (garethnunns) [reply website/versioned_docs/version-13.0/chases/chase-options.md:52]: I'll make this a global change

[the remark this answers is not in the document]
Comment (garethnunns) [reply website/versioned_docs/version-13.0/chases/chase-options.md]: Resolved by ad98f8b

File: docs/networking/connecting-the-arena-to-a-network.md [hunk 3]
```diff
@@ -12,21 +12,21 @@ Arena console.**
...
 into the Avolites TitanNet Switch (TNS) rack unit which provides
-standard wired ethernet connections. If you want to connect to a
+standard wired Ethernet connections. If you want to connect to a
 different optical switch you should contact Avolites for advice.
 
 Wired Ethernet Outputs
 ----------------------
 
-The Arena console has an in-built 1Gb TitanNet switch (TNS) that is
+The Arena console has an in-built 1GB TitanNet switch (TNS) that is
```
[the remark this answers is not in the document]
Comment (garethnunns) [reply docs/networking/connecting-the-arena-to-a-network.md]: Resolved in 3650ecf

**Review by Farrser — APPROVED** [review]

**Pull request merged**